### PR TITLE
275817 - Use i18n bay resource

### PR DESCRIPTION
**Pull request by MBausson**

### CODE REVIEW

File: app/views/rooms/_action_buttons.html.erb [hunk 1]
```diff
@@ -6,90 +6,90 @@
...
   <% if view_side==Frame.view_sides[:both] %>
      <%= content_tag :div, class: 'btn btn-default disabled' do %>
-      Vue avant & arrière
+      <%= t(".views.back_front") %>
     <% end %>
   <% else %>
     <%= link_to url_for(params.except(:controller, :action).permit(:view, :islet, :id, :bg, :format).merge({:view => (view_side == Frame.view_sides[:back] ? Frame.view_sides[:front] : Frame.view_sides[:back]), islet: params['islet'], id: params['id']})), class: 'btn btn-success', remote: true do %>
       <span class="glyphicon glyphicon-refresh" aria-hidden="true"></span>
-      <%= (view_side==Frame.view_sides[:back] ? 'Vue avant' : 'Vue arrière') %>
+      <%= (view_side==Frame.view_sides[:back] ? t(".views.front") : t(".views.back")) %>
```
Comment (nicolas-brousse) [inline app/views/rooms/_action_buttons.html.erb:27]: @MBausson Use Record attribute translation here. See this section on documentation https://guides.rubyonrails.org/i18n.html#translations-for-active-record-models (refer to roles example)

File: app/views/bays/new.html.erb [hunk 2]
```diff
@@ -1,11 +1,17 @@
-<%= render 'layouts/breadcrumb', breadcrumb_variables: {'Paramètres' => modeles_url, 'Baies et doubles-baies' => bays_path, 'Nouvelle baie' => ''} %>
+<%= render 'layouts/breadcrumb', breadcrumb_variables: {
+  t("breadcrumb.settings") => modeles_url,
+  t("breadcrumb.bays.index") => bays_path,
+  t("breadcrumb.bays.new") => ''}
+%>
```
Comment (nicolas-brousse) [inline app/views/bays/new.html.erb:5]:
```suggestion
<%= render "layouts/breadcrumb", breadcrumb_variables: {
  t("breadcrumb.settings") => modeles_url,
  t("breadcrumb.bays.index") => bays_path,
  t("breadcrumb.bays.new") => ""}
%>
```

**Review by nicolas-brousse — CHANGES_REQUESTED** [review]
@MBausson I've to check a bit more, but here is my first review

File: config/locales/fr.yml [hunk 3]
```diff
@@ -15,6 +15,55 @@ fr:
...
+      flashes:
+        created: La baie a été ajoutée.
+    edit:
+      title: Modifier la baie %{type} %{frames}
+    update:
+      flashes:
+        updated: La baie a bien été mise à jour.
+    destroy:
+      flashes:
+        destroyed: La bay bien été supprimé.
```
Comment (B-Rass) [inline config/locales/fr.yml:50]:
```suggestion
        destroyed: La baie bien été supprimé.
```

File: config/locales/fr.yml [hunk 4]
```diff
@@ -15,6 +15,55 @@ fr:
...
+    action_buttons:
+      caption:
+        ports: Ports
+        unreferenced_client: Le port correspondant de la carte jumelle est câblé
+        no_client: Le port correspondant de la carte jumelle n'est pas câblé
+      background_color: 'Couleur de fond : %{bg}'
+      background_choices:
+        modele: Modèle
+        manager: Gestionnaire
+        cluster: Cluster
```
Comment (B-Rass) [inline config/locales/fr.yml:61]: @nicolas-brousse Pourquoi ne pas directement utiliser les noms des modèles ici ?

**Review by B-Rass — CHANGES_REQUESTED** [review]

File: config/locales/activerecord.fr.yml [hunk 5]
```diff
@@ -220,6 +224,19 @@ fr:
         position: Position
         bay_id: Baie
         u: U
+      bay:
+        lane: Ligne
+        position: Position
+        materials_count:
+          one: 1 serveur
+          other: "%{count} serveurs"
+          zero: 0 serveurs
```
Comment (B-Rass) [inline config/locales/activerecord.fr.yml:233]:
```suggestion
          zero: 0 serveur
```

**Review by B-Rass — CHANGES_REQUESTED** [review]

**Review by nicolas-brousse — APPROVED** [review]

**Review by B-Rass — APPROVED** [review]

**Pull request merged**